capsules/object_speed/capsule.py

```diff
@@ -0,0 +1,32 @@
+from vcap import (
+    BaseCapsule,
+    NodeDescription,
+    DeviceMapper,
+    IntOption
+)
+
+from .backend import Backend
+from .stream_state import StreamState
+
+
+class Capsule(BaseCapsule):
+    name = "object_speed"
+    description = "✨ Measure pixel-per-second speed on tracked detections, " \
+                  "and puts the information in the extra_data field."
+    version = 1
+    device_mapper = DeviceMapper.map_to_single_cpu()
+    stream_state = StreamState
+    input_type = NodeDescription(
+        size=NodeDescription.Size.SINGLE,
```

Comment:
Fun fact, this is the first capsule we've made that doesn't have to specify object type